### ISSUE TITLE
Not sure on how to change the Role for each view?

### ISSUE THREAD

**Issue by bingimar** [carltongibson/neapolitan#32]
can someone explain how to change the Role for each view? I'd like to make some views only list and detail.

**Comment by carltongibson** [comment]
Hi @bingimar. 

I need to make this easier, or at least documented better, but the base approach would be to override `get_urls()`. Something like... 

```python
    @classonlymethod
    def get_urls(cls):
        verbose_name = cls.model._meta.model_name
        urlpatterns = [
            path(
                f"{verbose_name}/",
                cls.as_view(role=Role.LIST),
                name=f"{verbose_name}-list",
            ),
            path(
                f"{verbose_name}/<int:pk>/",
                cls.as_view(role=Role.DETAIL),
                name=f"{verbose_name}-detail",
            ),
        return urlpatterns
```

Will give you just the list and detail views.

**Comment by carltongibson** [comment]
Fixed in 24.4 You can now do something like: 

```
    urlpatterns = [
        *BookmarkView.get_urls(roles={Role.LIST, Role.DETAIL}),
    ]

```
